app/routes/agenda-items.ts

```diff
@@ -28,7 +30,7 @@ const getQuery = ({
     "session.governing-body.administrative-unit",
     "session.governing-body.administrative-unit.location",
   ].join(","),
-  municipality: municipality ? municipality : undefined,
+  //municipality: locationIds ? locationIds : undefined,
```

Comment:
**nitpick:** Can you remove commented code in order to keep code clean :pray:

Reply:
Done!